contrib/join-library/pom.xml

```diff
@@ -5,11 +5,33 @@
 
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>com.google.cloud.dataflow</groupId>
+  <groupId>org.linuxalert.dataflow</groupId>
   <artifactId>google-cloud-dataflow-java-contrib-joinlibrary</artifactId>
-  <name>Join library</name>
+  <name>Dataflow Join Library</name>
   <description>Library with generic join functions for Dataflow.</description>
-  <version>0.0.2-SNAPSHOT</version>
+  <url>https://github.com/GoogleCloudPlatform/DataflowJavaSDK/tree/master/contrib/join-library</url>
+  <developers>
+    <developer>
+      <id>magru</id>
+      <name>Magnus Runesson</name>
+      <email>M (dot) Runesson (at) gmail (dot) com</email>
+      <organization>Linuxalert</organization>
+      <organizationUrl>http://www.linuxalert.org/</organizationUrl>
+      <roles>
+        <role>Project-Administrator</role>
+        <role>Developer</role>
+      </roles>
+      <timezone>+1</timezone>
+    </developer>
+  </developers>
+  <contributors>
+    <contributor>
```

Comment:
can you please update this to match the `Authors.md` file? That has Google listed as well.

Would suggest copying [this section from root `pom.xml`](https://github.com/GoogleCloudPlatform/DataflowJavaSDK/blob/master/pom.xml#L50)